build/jobs/provision-deploy.yml

```diff
@@ -47,13 +47,15 @@ jobs:
      
         $additionalProperties = $flattenedCors
      
-        $additionalProperties["SqlServer__DeleteAllDataOnStartup"] = "true"
+        $additionalProperties["SqlServer__DeleteAllDataOnStartup"] = "false"
+        $additionalProperties["SqlServer__AllowDatabaseCreation"] = "true"
         $additionalProperties["ASPNETCORE_FORWARDEDHEADERS_ENABLED"] = "true"
 
         $templateParameters = @{
             fhirVersion = "${{ parameters.version }}"
             appServicePlanResourceGroup = "${{ parameters.appServicePlanResourceGroup }}"
```

Comment:
To make a app service plan the resource group shouldn't be passed. There is a condition that is currently missing from the docker scripts that is present in the old scripts to prevent creating a new app service if an app service resource group is passed. This is being put back in in PR #988.